.github/workflows/deploy.yml

```diff
@@ -28,3 +27,18 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.SHOPIFY_GH_ACCESS_TOKEN }}
+
+      - name: Set 'latest' NPM dist tag
+        if: steps.changesets.outputs.published == 'true' && github.ref_name == vars.LATEST_STABLE_VERSION
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: |
+          cat << EOF > "$HOME/.npmrc"
+            //registry.npmjs.org/:_authToken=$NPM_TOKEN
+          EOF
+          for pkg in $(echo "${{ steps.changesets.outputs.publishedPackages }}" | jq -r '.[] | @base64'); do
+            _jq() {
+              echo ${pkg} | base64 --decode | jq -r ${1}
+            }
+            npm dist-tag add $(_jq '.name')@$(_jq 'version') latest
+          done
```

Comment:
This (should) work. It uses the [output](https://github.com/changesets/action#outputs) variable  `publishedPackages` from the `changesets/action@v1` action.

i.e.:
```
[{"name": "@shopify/ui-extensions", "version": "2023.7.2"}, {"name": "@shopify/ui-extensions-react", "version": "2023.7.2"}]
```

I'm also crossing my fingers that github actions `ubuntu-latest` has `jq` installed by default. 

Haven't gotten around to testing this quite yet.

Reply:
@vividviolet I was able to test this update by mocking the publish step and attempting to add dist tags that already existed to prove the new step works. Should be good to go now and I updated the deploy instructions as well.